staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/sample_and_watermark_test.go

```diff
@@ -120,3 +120,105 @@ func getHistogramCount(registry compbasemetrics.KubeRegistry, metricName string)
 	}
 	return 0, errMetricNotFound
 }
+
+func BenchmarkSampleAndWaterMarkHistogramsVecEltSimple(b *testing.B) {
```

Comment:
FWIW - benchmarks can also go to a separate PR.

Reply:
They _could_ but I chose to include them here because the point is to show that the timing ratio histograms added in this PR are much more efficient than the sample-and-watermark histograms.

Reply:
The git commit message includes the benchmark results.